mergeSort.js

```diff
@@ -0,0 +1,17 @@
+function mergeSort(arr, sIndx, eIndx) {
+  let merged = [];
+  let mid = Math.floor(eIndx - sIndx / 2);
+  let j = mid + 1;
```

Comment:
The first part of the function should  define the left ,right and middle of the array. then use the merge function that take in the left,right and the array. The merge function is where comparing of the values is happening.